app/javascript/analytics/dashboard.js

```diff
@@ -118,6 +129,7 @@ function drawCharts(data, timeRangeLabel) {
 
   drawChart({
     id: 'reactions-chart',
+    showPoints: timeRangeLabel !== '',
```

Comment:
We do this same logic check for every `drawChart` in this function, so let's bring it out into a variable e.g.

```js
// When timeRange is "Infinity" we hide the points to avoid over-crowding the UI
const showPoints = timeRangeLabel !== '';
```

Then we can just reference that variable in each `drawChart` 🙂  (I've added a comment in the suggestion too, just to highlight why we're checking this.

Reply:
You're right! Thanks a lot